itou/www/apply/forms.py

```diff
@@ -845,6 +846,79 @@ def clean(self):
         JobSeekerProfile.clean_pole_emploi_fields(self.cleaned_data)
 
 
+class CertifiedCriteriaInfoRequiredForm(forms.ModelForm):
+    """API Particulier required information.
+    https://github.com/etalab/siade_staging_data/blob/develop/payloads/api_particulier_v2_cnav_allocation_adulte_handicape/200_beneficiaire.yaml
+    """
+
+    birth_place = forms.ModelChoiceField(
+        queryset=asp_models.Commune.objects,
+        label="Commune de naissance",
+        help_text="La commune de naissance est obligatoire lorsque le salarié est né en France. "
+        "Elle ne doit pas être renseignée s’il est né à l'étranger.",
+        widget=RemoteAutocompleteSelect2Widget(
+            attrs={
+                "data-ajax--url": reverse_lazy("autocomplete:communes"),
+                "data-ajax--cache": "true",
+                "data-ajax--type": "GET",
+                "data-minimum-input-length": 2,
```

Comment:
Autrement, les autochtones de la commune d’Y seront bloqués.
```suggestion
                "data-minimum-input-length": 1,
```

Reply:
[89 habitants recensés en 2021](https://fr.wikipedia.org/wiki/Y_(Somme)), mais soit. :) Je ne savais même pas que cette commune existait. Ypsiloniens est un joli nom d'ailleurs.